Oneirocom/Magick
# File drawer integration

## What Changed:

The documents and spell folders contain updated and accurate data in their respective directories.

## How to test:

Go to the new left sidebar implementation and navigate to the document/spell folder. Verify if there is content present there, including documents with uploaded knowledge and spells that have been created.

## Additional information:

https://github.com/Oneirocom/Magick/assets/55635977/e8a997ea-ad9f-4c48-9f36-c126d3aaeb49
